src/utils/http.ts

```
@@ -0,0 +1,48 @@
+import axios, {AxiosInstance} from 'axios'
+
+
+export const BASE_URL = 'https://api.app.shortcut.com/api/v3'
+const VERSION_PATH = '/api/v3'
+
+/**
+ * Create a pre-authenticated HTTP client. All request paths are relative to {@link BASE_URL}.
+ * @param apiKey - The Shortcut API token to authenticate requests with
+ */
+export function createHttpClient(apiKey: string): AxiosInstance {
```

ESLint

[error] Declare this without inline export and list it in a single export statement at the end of the file.

(yenz/export-at-end-of-file)